### DISCUSSION TITLE
Nepharite Warlord (ALGEROTH) [R]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#13] [Card Balance]
Digital card game :
Nepharite Warlord (ALGEROTH) [R]
NEPHARITE
Your Undead Legionnaires gain +1 to [F] and [A] .
F: 9 S: 6 A: 5 V: 9

My suggestion :
NEPHARITE
Your other non-UNIQUE ALGEROTH warriors have their cost reduce by 1 [DP] and gain +1 to F and A.
F: 9 S: 6 A: 8 V: 10

We have cards for NEPHARITE :
-necrovisual link
-portal of  undeath
-azogar polarm 

If we update "Nepharite warlord" and "Nepharite of algeroth", we could build a nepharite deck.  :arrow_right:  more variety :sunglasses:

**Comment by TomekPL-1** [comment]
Nepharite Warlord (Dark Legion) [R]
NEPHARITE. At the end of your turn add one UNDEAD LEGIONNAIRE from your discard pile in to the SQUAD
F: 7 S: 4 A: 5 V: 7

**Reply by framerate** [reply]
What if we removed the unlimited keyword from the undead, since they are the only ones in the whole game that have it

Then we could make one of the nepharites create undead tokens

**Reply by TomekPL-1** [reply]
Would be cool ability Justin.

What about Undead Legionnaire HORDE project? ;)

**Reply by framerate** [reply]
I'm not opposed! I never liked UNLIMITED since it was a one off. 

Hoarde might make them fun

**Reply by 21-ric** [reply]
It's really interesting. I like the idea.

**Reply by TomekPL-1** [reply]
HORDE sounds even more fun when Undead Legionnaire do not have UNLIMITED (only 3 in deck) and need to be generated by other cards like Nepharite Warlords

**Reply by framerate** [reply]
Tomek-

Can you post any notes you have for horde mode for other people
